### ISSUE TITLE
Remove Products.CMFFormController dependency.

### ISSUE THREAD

**Issue by tisto** [plone/Products.CMFPlone#448]
http://jenkins.plone.org/view/Dependencies/job/plone-package-dependencies-cmf/36/artifact/deps/Products.CMFFormController.txt

```
Products/CMFPlone/factory.py:                u'Products.CMFFormController:CMFFormController',
Products/CMFPlone/setuphandlers.py:            'CMFFormController', 'Products.CMFFormController',
plone/app/upgrade/v30/tests.py:from Products.CMFFormController.interfaces import IFormControllerTool
plone/app/upgrade/v30/alphas.py:from Products.CMFFormController.interfaces import IFormControllerTool
plone/app/testing/layers.py:        ('Products.CMFFormController',           {'loadZCML': True}, ),
```

**Comment by vangheem** [comment]
Seems like as long as we're checking ATContentTypes around, we can not remove CMFFormController in Plone 5. This will have to wait for Plone 5.1 when we do not ship with ATContentTypes.

**Comment by tomgross** [comment]
The following core packages depend on CMFFormController:

 - [ ] CMFEditions
 - [ ] ATContentTypes
 - [ ] CMFPlone (login!)
 - [ ] PasswordResetTool
 - [ ] Archetypes

All non AT dependencies could be removed earlier. What's the state of plone.login BTW?

**Comment by jensens** [comment]
This was addressed in the Plone 5.2 python3 migration and is removed already there (branch python3)
